qa/rpc-tests/abc-monolith-activation.py

```diff
@@ -49,11 +51,12 @@ def create_and_tx(self, count):
         tx.vout = []
         for _ in range(count):
             tx.vout.append(CTxOut(value, CScript([OP_1, OP_1, OP_AND])))
-        tx_signed = node.signrawtransaction(ToHex(tx))["hex"]
+        tx_signed = node.signrawtransaction(ToHex(tx),None,None,"ALL|FORKID")["hex"]
         return tx_signed
 
     def run_test(self):
         self.test = TestManager(self, self.options.tmpdir)
+        print ("num nodes is " + str(len(self.nodes)))
```

Comment:
I would remove this print statement before merging

Reply:
done